replayer/replayer.cpp

```diff
@@ -25,21 +25,18 @@ std::ostream& operator<<(std::ostream& out, const Replayer& o) {
   auto width = o.map.width;
   auto data = o.map.data.data();
 
-  if (o.keyframe != 0)
-    out << 0 << " " << o.keyframe << " ";
+  if (o.keyFrameInterval != 0)
+    out << 0 << " " << o.keyFrameInterval << " ";
   out << height << " " << width << " ";
   out.write((const char*)data, height * width); // Write map data as raw bytes
 
-  auto kf = o.keyframe == 0 ? 1 : o.keyframe;
+  auto kf = o.keyFrameInterval == 0 ? 1 : o.keyFrameInterval;
```

Comment:
I originally meant to rename the actual `kf` variable here, since that makes the loop below more readable and would make the previous comments obsolete.